packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tabs/Tabs.js

```diff
@@ -0,0 +1,16 @@
+import styled from 'styled-components'
+import { Tabs } from 'grommet'
+
+import { borderColor } from './colors'
+
+const StyledTabs = styled(Tabs)`
```

Comment:
It looks like `Tabs` uses `Box` under the hood, so you should be able to use Box's border prop to define the borders. https://github.com/grommet/grommet/blob/NEXT/src/js/components/Tabs/Tabs.js